Sage/carbon
# docs: create new static welcome page

### Proposed behaviour


There should be a static `Welcome` page that replicates the look and feel of the current demo site and is served first on opening of `storybook`. The Welcome story should be the first story and the rest should be sorted alphabetically.

### Current behaviour


There is no welcome landing page and the stories are not sorted alphabetically

### Checklist


<del>- [ ] Unit tests<del>
<del>- [ ] Cypress automation tests<del>
<del>- [ ] Typescript `d.ts` file added or updated<del>

### Additional context

This is the first PR to remove the demo site and fix the issue with loading fonts via CDN. The next step is to add a `webpack` config and remove the `carbon-factory` dependency.

FE-2525 has also been fixed as part of this PR

### Testing instructions

Open storybook and ensure Welcome is the first story and is opened first. Check the links and buttons work as expected


![image](https://user-images.githubusercontent.com/44157880/74842808-dbcf2280-5322-11ea-8e6d-1d34a602169b.png)
